packages/ibm-products/src/components/ConditionBuilder/sampleInput.tsx

```diff
@@ -0,0 +1,241 @@
+import {
```

Comment:
same comment as above about moving this into a separate folder that contains sample / storybook assets.

Reply:
modified as advised